modular_skyrat/modules/loadouts/loadout_items/loadout_datum_gloves.dm

```diff
@@ -88,7 +88,7 @@ GLOBAL_LIST_INIT(loadout_gloves, generate_loadout_items(/datum/loadout_item/glov
 
 /datum/loadout_item/gloves/armwraps
 	name = "Colourable Arm Wraps"
-	item_path = /obj/item/clothing/gloves/bracer/wraps
+	item_path = /obj/item/clothing/gloves/wraps //BUBBER EDIT (removed "bracer" path before "wraps")
```

Comment:
Can be modularised with an override

Reply:
Also doesnt have to be done if you make an override of the `/obj/item/clothing/gloves/bracer/wraps` armor values